src/app/lorax/directives/share.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * @fileOverview Share directive
+ * @author <a href="mailto:chris@work.co">Chris James</a>
+ * @author <a href="mailto:sneethling@mozilla.com">Schalk Neethling</a>
+ */
+define(['angular', 'jquery'], function (angular, $) {
+    'use strict';
+
+    /**
+     * Share directive
+     */
+    var ShareDirective = function () {
+        return {
+            restrict: 'A',
+            scope: true,
+            controller: ShareController
+        };
+    };
+
+    /**
+    * Controller for Share directive
+    * @constructor
+    */
+    var ShareController = function (
+        $scope,
+        dataService,
+        routesService
+    ) {
+
+        console.log('ShareController');
```

Comment:
Perhaps a fresh re-start might open up some ideas to what the original problem was.